packages/suite-desktop-core/src/modules/bridge.ts

```diff
@@ -24,9 +30,28 @@ const handleBridgeStatus = async (
     return status;
 };
 
+const start = async (bridge: BridgeProcess | TrezordNode) => {
+    if (bridgeNode) {
```

Comment:
Maybe unify `start`, `startDev` & `startTest` in `TrezorNode` and remove the first `if` branch?

Reply:
[3be0563](https://github.com/trezor/trezor-suite/pull/9985/commits/3be0563650ad484325f16de6a670ac9b560eeda3) and [41b0df0](https://github.com/trezor/trezor-suite/pull/9985/commits/41b0df0cc53bc923e3e66f5d2ea488ae509c86af)